From 6eda71b23c2a7c1999c37aeb1d2eb99456ba52c8 Mon Sep 17 00:00:00 2001
From: Matt Koenig <der_koenig@rocketmail.com>
Date: Sun, 16 Feb 2014 17:23:02 +0100
Subject: [PATCH] Fix LPAPlayer build for msm7x27a

---
 media/libstagefright/Android.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/media/libstagefright/Android.mk b/media/libstagefright/Android.mk
index 8fd8dd1..e5237b1 100644
--- a/media/libstagefright/Android.mk
+++ b/media/libstagefright/Android.mk
@@ -95,6 +95,10 @@ ifneq ($(filter caf bfam,$(TARGET_QCOM_AUDIO_VARIANT)),)
             LOCAL_SRC_FILES += LPAPlayer.cpp
             LOCAL_CFLAGS += -DLEGACY_LPA -DUSE_LPA_MODE
         endif
+        ifeq ($(call is-chipset-in-board-platform,msm7x27a),true)
+            LOCAL_SRC_FILES += LPAPlayer.cpp
+            LOCAL_CFLAGS += -DLEGACY_LPA -DUSE_LPA_MODE
+        endif
         ifeq ($(NO_TUNNEL_MODE_FOR_MULTICHANNEL),true)
             LOCAL_CFLAGS += -DNO_TUNNEL_MODE_FOR_MULTICHANNEL
         endif
-- 
1.8.5.5
