From 4c1df63147f3518eddb0c46f7be4ab01fcc244f6 Mon Sep 17 00:00:00 2001
From: joebonrichie <josephriches@gmail.com>
Date: Tue, 28 Jan 2014 17:38:53 +0000
Subject: [PATCH] include:audiopolicymanagerbase: fix compile issue

---
 include/hardware_legacy/AudioPolicyManagerBase.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/hardware_legacy/AudioPolicyManagerBase.h b/include/hardware_legacy/AudioPolicyManagerBase.h
index 5ba09f1..ada176c 100644
--- a/include/hardware_legacy/AudioPolicyManagerBase.h
+++ b/include/hardware_legacy/AudioPolicyManagerBase.h
@@ -149,6 +149,7 @@ public:
         virtual status_t dump(int fd);
 
         virtual bool isOffloadSupported(const audio_offload_info_t& offloadInfo);
+	void handleNotificationRoutingForStream(AudioSystem::stream_type stream);
 
 protected:
 
@@ -582,7 +583,6 @@ private:
                 int indexInUi);
         // updates device caching and output for streams that can influence the
         //    routing of notifications
-        void handleNotificationRoutingForStream(AudioSystem::stream_type stream);
 };
 
 };
-- 
1.8.3.2

